### PR TITLE
Added title to the home page and made it responsive

### PR DESCRIPTION
**Pull request by ojasaklechayt**
Issue Number: #49 

Description:
      I have done the following things:
        1. Added H1 tag which says " Welcome to Jarvis " which improves the UI
        2. Made it basic responsive
 
Screenshot: 
image

### CODE REVIEW

**Comment by J0SAL** [comment]
@Abhijit25Mishra can you review this

**Comment by Abhijit25Mishra** [comment]
yeah sure @J0SAL

**Comment by Abhijit25Mishra** [comment]
- Make your PR title specific (if possible)
- add your ./frontend/package-lock.json into .gitignore file  
- you added an empty file Login.css, if this is not begin used then remove it.
- remove the ./package.json from the root folder. why is this even there ? 
- Update your PR template like this:
> 
> Fixes: #99
> Description:
> Attach Screenshots (if needed)
>

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]
- Make your PR title specific (if possible)
- add your ./frontend/package-lock.json into .gitignore file  
- you added an empty file Login.css, if this is not begin used then remove it.
- remove the ./package.json from the root folder. why is this even there ? 
- Update your PR template like this:
> 
> Fixes: #99
> Description:
> Attach Screenshots (if needed)
>

**Comment by ojasaklechayt** [comment]
> * Make your PR title specific (if possible)
> * add your ./frontend/package-lock.json into .gitignore file
> * you added an empty file Login.css, if this is not begin used then remove it.
> * remove the ./package.json from the root folder. why is this even there ?
> * Update your PR template like this:
> 
> > Fixes: #99
> > Description:
> > Attach Screenshots (if needed)

Done Ser

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]
package-lock.json is still added. please remove that.

you can either delete that from the root folder or create another gitignore file in the root folder and add that file there.

File: frontend/.gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
```
Comment (Abhijit25Mishra) [inline frontend/.gitignore:37]: add package-lock.json as well.

Comment (ojasaklechayt) [reply frontend/.gitignore:37]: Done ser

**Comment by Abhijit25Mishra** [comment]
see, package-lock.json is still commited. please uncommit that.

have a look through this if you dont know how to. 
(https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/uncommit-git-last-commit-file-changes-pushed-deleted-message)

**Comment by ojasaklechayt** [comment]
> see, package-lock.json is still commited. please uncommit that.
> 
> have a look through this if you dont know how to. (https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/uncommit-git-last-commit-file-changes-pushed-deleted-message)

I have done it.

File: frontend/package-lock.json
Comment (Abhijit25Mishra) [inline frontend/package-lock.json:1]: Your not supposed to delete this. only add this file to ur git ignore file.

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]

**Comment by Abhijit25Mishra** [comment]
also Update your PR template like this:

    Fixes: #99
    Description:
    Attach Screenshots (if needed)

**Review by J0SAL — CHANGES_REQUESTED** [review]
@ojasaklechayt Please resolve the issues with deployment, I think it is failing as you've deleted the package.json file

**Comment by ojasaklechayt** [comment]
> @ojasaklechayt Please resolve the issues with deployment, I think it is failing as you've deleted the package.json file

which package json fine ? root one ?

**Comment by ojasaklechayt** [comment]
I tried doing this but I am not able to fix it.

**Comment by J0SAL** [comment]
@ojasaklechayt , On your local machine first run `npm run build` and you'll be able to see what the error is about

**Comment by ojasaklechayt** [comment]
> @ojasaklechayt , On your local machine first run `npm run build` and you'll be able to see what the error is about

Thankyou so much!! It is correctly deployed now. I thought it was some package error but it turns out to be syntax error in Login.jsx page.

**Comment by ojasaklechayt** [comment]
@J0SAL

**Review by J0SAL — CHANGES_REQUESTED** [review]
image
Can you update the alignment, 
keep the image on the left, and the title on the right? 
The button should be just below the title

**Comment by ojasaklechayt** [comment]
> image Can you update the alignment, keep the image on the left, and the title on the right? The button should be just below the title

Done

**Review by J0SAL — CHANGES_REQUESTED** [review]
It's not responsive, please work on these, 
image

In mobile view, there shouldn't be a need of a scrollbar for the login page

**Comment by ojasaklechayt** [comment]
> It's not responsive, please work on these, image
> 
> In mobile view, there shouldn't be a need of a scrollbar for the login page

Done Ser

File: frontend/components/common/Login.tsx [hunk 2]
```diff
@@ -5,27 +5,44 @@ function Login() {
   return (
     <div>
       <div
+        className="inner-home"
```
Comment (J0SAL) [inline frontend/components/common/Login.tsx:8]: where is inner-home implemented?

File: frontend/components/common/Login.tsx [hunk 3]
```diff
@@ -5,27 +5,44 @@ function Login() {
...
           alt="no data"
-          width={"100vh"}
-          height={"100vh"}
+          width={"300vh"}
+          height={"300vh"}
+          style={{paddingTop:"5vh"}}
         />
-        <ConnectButton moralisAuth={false} />
+        <div
+          className="outter-home"
```
Comment (J0SAL) [inline frontend/components/common/Login.tsx:28]: where is outer-home implemented?

**Review by J0SAL — CHANGES_REQUESTED** [review]
image

remove this gap, try to bring the image and text in the center

it should look like this, 
image

File: frontend/components/common/Login.tsx (hunk 2 above)
Comment (ojasaklechayt) [reply frontend/components/common/Login.tsx:8]: I was trying to use custom css but I guess it was not required. Hence I will remove that.

File: frontend/components/common/Login.tsx (hunk 3 above)
Comment (ojasaklechayt) [reply frontend/components/common/Login.tsx:28]: Same as inner-home implemented

**Comment by ojasaklechayt** [comment]
> image
> 
> remove this gap, try to bring the image and text in the center
> 
> it should look like this, image

Done

**Comment by J0SAL** [comment]
@ojasaklechayt resolve the merge conflicts, everything else looks good 👍

**Comment by ojasaklechayt** [comment]
I will create another pr

**Pull request closed without merging**